packages/components/src/__future__/Tag/_docs/Tag.stories.tsx

```diff
@@ -1,94 +1,52 @@
 import React from "react"
-import { Meta, StoryFn } from "@storybook/react"
-import { TagIcon } from "~components/Icons"
-import { ComponentDocsTemplate } from "~storybook/components/DocsContainer"
-import { StickerSheet } from "~storybook/components/StickerSheet"
-import { Tag } from ".."
+import { Meta, StoryObj } from "@storybook/react"
+import { AcademyIcon, ActionOffIcon, AddIcon, TagIcon } from "~components/Icons"
+import { Tag, TagColorKeys } from ".."
+import styles from "./TagStories.module.scss"
 
 export default {
```

Comment:
Create a `Story` type within this file (eg. in `WorkflowFooter`), then add any default args for your stories into the `args` prop in the `meta`.